Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -802,12 +807,14 @@ function InvokeTestAgentConfigExe([string[]] $Arguments, [string] $Version, [Sys
     $ExeName = "TestAgentConfig.exe"
     if (-not (Test-IsAdmin))
     {
+        Write-Host "##vso[task.logissue type=error;code=001005;]"
         throw "You need to be an Administrator to run this tool."
     }
 
     $vsRoot = Locate-TestVersionAndVsRoot($Version)
     if ([string]::IsNullOrWhiteSpace($vsRoot))
     {
+        Write-Host "##vso[task.logissue type=error;code=001003;]"
```

Comment:
1003 is being used at many places ... although in most situations the root cause could be the same and so it makes sense to reuse the same code, but in some cases it may not be the same reason ... and in that case we may want to distinctly know where the error came from.